dataRetriever/factory/resolverscontainer/metaResolversContainerFactory.go

```diff
@@ -308,13 +309,25 @@ func (mrcf *metaResolversContainerFactory) generateTrieNodesResolvers() error {
 	resolversSlice := make([]dataRetriever.Resolver, 0)
 
 	identifierTrieNodes := factory.AccountTrieNodesTopic + core.CommunicationIdentifierBetweenShards(core.MetachainShardId, core.MetachainShardId)
-	resolver, err := mrcf.createTrieNodesResolver(
-		identifierTrieNodes,
-		triesFactory.UserAccountTrie,
-		0,
-		mrcf.numTotalPeers,
-		core.MetachainShardId,
-	)
+
```

Comment:
Maybe we can move L313-L330 into baseResolverContainerFactory since we have mostly the same thing in shardResolversContainerFactory L241-L258?

Reply:
indeed, updated, thanks